src/research/research-tools-resources.md

```diff
@@ -75,7 +75,6 @@ User researchers are also encouraged to look outside of the organisation for exi
 - [Laws of UX](https://lawsofux.com/)
 - [Open Design Kit from Bocoup](http://opendesignkit.org/)
 - [Service Design Toolkit](http://servicedesigntoolkit.org/)
-- [Design Research Techniques from CFC Mediala](http://designresearchtechniques.com/)
```

Comment:
Has this been checked with someone from research team if can be removed?

Reply:
Removal approved by Chris H